front/cypress/e2e/idea_posting_anonymous.cy.ts

```diff
@@ -1,121 +1,8 @@
 import moment = require('moment');
 import { randomEmail, randomString } from '../support/commands';
 
-describe('Continuous ideation with anonymous participation allowed', () => {
```

Comment:
I thought the added value of a near copy of the other (more commonly used timeline projects) is low, so let's speed up our E2E test suite instead. Or was there a particular reason to have both continuous and timeline tests? :)

Reply:
I believe we had some bug which only impacted continuous projects :thinking: I think it's valuable to have both since timeline and continuous projects are two separate things in our platform, and they can be impacted by changes in different ways?